packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -268,12 +255,18 @@ export function TabList<T>(props: SpectrumTabListProps<T>) {
     setTabListState(state);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [state.disabledKeys, state.selectedItem, state.selectedKey, props.children]);
-  let stylePropsForVertical = orientation === 'vertical' ? styleProps : {};
+
+  let collapseStyle : React.CSSProperties = collapsed && orientation !== 'vertical' ? {maxWidth: 'calc(100% + 1px)', overflow: 'hidden', visibility: 'hidden', position: 'absolute'} : {maxWidth: 'calc(100% + 1px)'};
+  let stylePropsFinal = orientation === 'vertical' ? styleProps : {style: collapseStyle};
+
+  if (collapsed && orientation !== 'vertical') {
```

Comment:
question for second reviewer, should we move this into our aria hook? it seems very specific to our implementation in rendering, so I think it's fine here

Reply:
I agree, the collapse behavior is specific to Spectrum, I'd rather not force these styles on aria hook consumers.